packages/rum-core/src/common/utils.js

```diff
@@ -324,6 +324,22 @@ function removeInvalidChars(key) {
   return key.replace(/[.*"]/g, '_')
 }
 
+/**
+ * Removes the username:password portion from Urls
+ *
+ * Ex: https://a:b@example.com/c  -> https://example.com/c
```

Comment:
Maybe we should replace the redacted parts with `[REDACTED]`, e.g. 

`https://[REDACTED]:[REDACTED]@example.com/c`

I think the node agent does this, I'm not sure if they use the exact format of `[REDACTED]` though.

Reply:
Browsers convert the request of this form `https://a:b@example.com/c  -> https://example.com/c` while using XHR. Its better to align with browser and get rid of the userpass information than folllowing Node.

Reply:
I think this is different compared to what browsers do. We're using the URL here to show as the span name. We should show something as close to what they used in their code as possible. In other words, It is important to know whether the username and password were redacted or they simply were not there in the first place. If the user only sees `https://example.com/c` they can't know if there was any username and password.

Reply:
Done